scrapers/EuroPornstar/EuroPornstar.yml

```diff
@@ -0,0 +1,181 @@
+name: 06_europornstar
```

Comment:
Do you mind chacning the name to reflect `EuroPornstar` ?

Reply:
done, yeah my bad, that was the testing name I was using before copying it here, fixed it just now (I am used to codeberg and gitea so the process here is a bit new to me).